packages/template-retail-react-app/app/pages/product-list/index.test.js

```diff
@@ -82,6 +82,20 @@ const MockedComponent = ({isLoading, isLoggedIn = false, searchQuery}) => {
                     </div>
                 )}
             />
+            <Route
```

Comment:
There's no need to add a new `<Route />`, we can add the `'/search'` path to the existing `<Route />`:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/036a4499f56d02ee8db168cac9a4c842e8f45185/packages/template-retail-react-app/app/pages/product-list/index.test.js#L72

```
                path={[
                    createPathWithDefaults('/category/:categoryId'),
                    createPathWithDefaults('/search')
                ]}
```